HRSolutions/src/main/java/org/hrSolution/model/CompanyRegistrationModel.java

```diff
@@ -0,0 +1,49 @@
+package org.hrSolution.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+import org.hrSolution.model.PreEmployeeRegistrationModel.PreEmployeeRegistrationBuilder;
+
+public class CompanyRegistrationModel {
+
+	@JsonProperty("cName")
+	private String companyName;
+	@JsonProperty("cDesc")
+	private String companyDescription;
+
+	public CompanyRegistrationModel(CompanyRegistrationModelBuilder companyRegistrationModelBuilder) {
+		this.companyName = companyRegistrationModelBuilder.companyName;
+		this.companyDescription = companyRegistrationModelBuilder.companyDescription;
+	}
+
+	public String getCompanyName() {
+		return companyName;
+	}
+
+	public String getCompanyDescription() {
+		return companyDescription;
+	}
+
+	public static class CompanyRegistrationModelBuilder {
```

Comment:
Write comment about this class as below:
Builder class of  CompanyRegistraionModel